### PR TITLE
add German lang translation

### PR DESCRIPTION
**Pull request by ClientCrash**
Added a translation for the german language.

### CODE REVIEW

File: res/lang/de.ini [hunk 1]
```diff
@@ -0,0 +1,45 @@
+capslock = Feststelltaste
+err_alloc = Speicherzuweisung fehlgeschlagen
+err_bounds = Listenindex ist außerhalb des Bereichs
+err_chdir = Fehler beim oeffnen des home-ordners
+err_console_dev = Zugriff auf die Konsole fehlgeschlagen
+err_dgn_oob = log message
```
Comment (AnErrupTion) [inline res/lang/de.ini:6]: Please also translate this.

File: res/lang/de.ini [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+err_perm_user = Fehler beim heruntersetzen der Nutzer Berechtigungen
+err_pwnam = Holen der Benutzerinformationen fehlgeschlagen
+err_user_gid = Fehler beim setzen der Gruppen Id des Nutzers
+err_user_init = Initialisierung des Nutzers fehlgeschlagen
+err_user_uid = Setzen der Benutzer Id fehlgeschlagen
+err_xsessions_dir = Fehler beim finden des Sitzungsordners
+err_xsessions_open = Fehler beim öffnen des Sitzungsordners
+f1 = F1 Herunterfahren
+f2 = F2 Neustarten
+login = Anmelden:
```
Comment (AnErrupTion) [inline res/lang/de.ini:39]: Please remove the semicolon.

File: res/lang/de.ini [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+logout = Abgemeldet
+numlock = Numtaste
+password = Passwort:
```
Comment (AnErrupTion) [inline res/lang/de.ini:42]: Same here.

**Review by AnErrupTion — CHANGES_REQUESTED** [review]

**Comment by ClientCrash** [comment]
Thanks. I will implement the changes shortly.

**Pull request merged**